.kokoro/populatesecrets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Have you used `git updateindex --chmod=+x .kokoro/populatesecrets.sh` to make sure this is executable?

Reply:
Oh no, doing so now, thanks for the catch. Although I've been running from my branch...